README.md

```diff
@@ -50,23 +48,51 @@ The following are the available scripts:
 
 In the project directory, you can run:
 
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:8000](http://localhost:8000) to view it in the browser.
-
 ### `npm run storybook`
+
 Runs the storybook
 
 ### `npm run build`
 
 Builds the app for production to the `build` folder.\
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
-## 5. Folder Structure
+### `npm run chromatic`
+
+Deploys the storybook to chromatic \
+Create .npmrc file and add projectToken=$project_token
```

Comment:
I think we should add more details like how and where to generate them, also add screenshots of your file as sample, can put dummy text for tokens in screenshots